package/common_init/overlay/lib/mdev/automount.sh

```diff
@@ -44,6 +44,10 @@ do_mount()
     rmdir $mountpoint
     exit 1
   fi
+  if ! chgrp extio $mountpoint ; then
```

Comment:
This will just change the permissions of the mountpoint directory, we'll also need to specify a mount option for all of the files to be accessible to a specific group.  See "Mount options for FAT" here: https://linux.die.net/man/8/mount -- the uid= and gid= options will control this.  For F2FS, since it has unix permission support we probably don't need any extra mount options.